staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -272,21 +284,17 @@ func (c *celAdmissionController) Validate(
 				continue
 			}
 
-			var param runtime.Object
-
-			// versionedAttributes will be set to non-nil inside of the loop, but
-			// is scoped outside of the param loop so we only convert once. We defer
-			// conversion so that it is only performed when we know a policy matches,
-			// saving the cost of converting non-matching requests.
-			var versionedAttr *admission.VersionedAttributes
+			var params []runtime.Object
```

Comment:
Would it help readability to have most of the new code added here put into a utility function that finds all the params for the given paramRef/namespaceParamRef settings? (I'm okay with deferring to a post-freeze cleanup task if this would be disruptive to this PR)

Reply:
refactored